docs/buildah-rmi.md

```diff
@@ -43,5 +43,11 @@ buildah rmi --force imageID
 
 buildah rmi imageID1 imageID2 imageID3
 
+## Files
+
+**registries.conf** (`/etc/containers/registries.conf`)
+
+registries.conf is the configuration file which specifies which registries should be consulted when completing image names which do not include a registry or domain portion.
```

Comment:
How about "which _container_ registries should be consulted" ?

Reply:
hmmm, wished I thought of that this weekend.  Good point, new pr in a bit @pixdrift thx for the suggestion.